appserver/jdbc/jdbc-runtime/src/main/java/org/glassfish/jdbcruntime/deployment/annotation/handlers/DataSourceDefinitionHandler.java

```diff
@@ -48,6 +48,7 @@
 import org.glassfish.apf.AnnotationInfo;
 import org.glassfish.apf.AnnotationProcessorException;
 import org.glassfish.apf.HandlerProcessingResult;
+import org.glassfish.config.support.TranslatedConfigView;
```

Comment:
You *could* import static `TranslatedConfigView.expandValue` so you only use `expandValue()` so it could be more concise :)